app/Http/Actions/BuildAchievementOfTheWeekDataAction.php

```diff
@@ -53,7 +34,8 @@ public function execute(?StaticData $staticData): ?EventAchievementData
             'sourceAchievement.game.badgeUrl',
             'sourceAchievement.game.system.iconUrl',
             'sourceAchievement.game.system.nameShort',
-            'forumTopicId',
+            'event',
+            'event.legacyGame',
```

Comment:
Should this be `event.game`? I see you have a TODO: rename there, but as of right now, they don't match.

Reply:
In latest, I went ahead and implemented the TODO. This is just so the relation name and DTO field closely match the column name. Should also make it easier to find these references later.